test/types/cptr/cross_type_cast.good

```diff
@@ -1,9 +1,9 @@
-cross_type_cast.chpl:13: warning: Casting c_ptr to a non-equivalent, non-char element type ('int(32)' -> 'real(32)') can cause undefined behavior.
-cross_type_cast.chpl:14: warning: Casting c_ptr to a non-equivalent, non-char element type ('int(32)' -> 'real(64)') can cause undefined behavior.
-cross_type_cast.chpl:28: warning: Casting c_ptrConst to a non-equivalent, non-char element type ('int(32)' -> 'real(32)') can cause undefined behavior.
-cross_type_cast.chpl:29: warning: Casting c_ptrConst to a non-equivalent, non-char element type ('int(32)' -> 'real(64)') can cause undefined behavior.
-cross_type_cast.chpl:35: warning: Casting c_ptr to a non-equivalent, non-char element type ('int(32)' -> 'real(32)') can cause undefined behavior.
-cross_type_cast.chpl:36: warning: Casting c_ptrConst to a non-equivalent, non-char element type ('int(32)' -> 'real(32)') can cause undefined behavior.
-cross_type_cast.chpl:42: warning: Casting c_ptr to a non-equivalent, non-char element type ('c_ptr(int(32))' -> 'c_ptr(real(32))') can cause undefined behavior.
+cross_type_cast.chpl:13: warning: Casting c_ptr to a pointer of non-equivalent, non-char element type ('c_ptr(int(32))' -> 'c_ptr(real(32))') can cause undefined behavior.
```

Comment:
Something about the new warning wording seems off because e.g. `c_ptr(int(32))` is not an element type

Reply:
My thinking was it is "a pointer of non-equivalent, non-char element type", and then we print the pointer types. What kind of phrasing change would you suggest? I think we still need to mention "element type" or something similar, as that's the "non-equivalent, non-char" thing we're complaining of.

Reply:
Here is a try:

```
warning: This cast from 'c_ptr(int(32))' to 'c_ptr(real(32))' casts a c_ptr to a pointer of non-equivalent, non-char element type and so can cause undefined behavior.`
```